compute_tools/src/compute.rs

```diff
@@ -235,7 +235,7 @@ impl ComputeNode {
 
     // Get basebackup from the libpq connection to pageserver using `connstr` and
     // unarchive it to `pgdata` directory overriding all its previous content.
-    #[instrument(skip(self, compute_state))]
+    #[instrument(skip_all)]
```

Comment:
Well, this was not a good change, should restore the `lsn`.

Reply:
fixed in [577f608](https://github.com/neondatabase/neon/pull/4584/commits/577f60893a9c84216f1345289109e222a8168b39)